rally/README.md

```diff
@@ -0,0 +1,68 @@
+# Rally
+
+A Flutter sample app based on the Material study Rally (a hypothetical, personal finance app). It
+showcases custom tabs, custom painted widgets, and custom animations.
+
+For info on the Rally Material Study, see: https://material.io/design/material-studies/rally.html
+
+**This sample is not currently in a finished state. We're in the process
+of building it out.**
+
+## Goals
+
+* Show how to customize a tab bar.
+* Show how to create reusable custom widgets with composition and custom painting.
+* Show how to create an app with tabs and child navigation screens.
+
+## Widgets / APIs
+
+* AccountsPage
+* BillsPage
+* BudgetsPage
+* Colors
+* Constants
+* DetailsPage
+* FinancialEntity
+* FinancialEntityCard
+* FinancialEntityDetails
+* FinancialEntityView
+* Formatters
+* HomePage
+* LoginPage
+* OverviewPage
+* RallyLineChart
+* RallyPieChart
+* SettingsPage
+* VerticalFractionBar
+
+## The important bits
+
+### `/charts/*`
+
+These are the custom charts. The circle chart and line chart are custom painters,
+while the single vertical bar chart is a simple composition of boxes.
+
+### `/sections/*`
+
+These are the main sections for the tab views and the child screen with the details and line chart.
+The financial entity is a reusable screen that is the base for the accounts, bills, and budgets
+screens.
+
+#### Notice
+
+* Theming is passed as a parameter in the constructor of `MaterialApp` (`theme:`).
```

Comment:
I don't know that this section is necessary here. We do all of these things in other sample apps, and you can always draw a little extra attention to them with inline comments in the code.

Reply:
Done